src/main/java/com/google/devtools/build/lib/rules/python/PythonSemantics.java

```diff
@@ -60,6 +60,11 @@ void collectDefaultRunfilesForBinary(RuleContext ruleContext, Runfiles.Builder b
       RuleContext ruleContext, Collection<Artifact> sources, PyCommon common);
 
   /**
+   * @return - {@link PathFragment} representing the python package.
+   */
+  PathFragment getPackageFragment(RuleContext ruleContext);
```

Comment:
Likewise, there is no need to put this here; since it's just PackageIdentifier#getRunfilesPath(), let's just use that and not pollute the PythonSemantics interface (which shouldn't exist anyway...)